v2/internal/util/metrictree/parser.go

```diff
@@ -204,6 +204,9 @@ func isValidIdentifier(s string) bool {
 		case el >= '0' && el <= '9':
 		case el == '-' || el == '_':
 		case el == '.':
+		case el == '+':
+		case el == '&':
+		case el == '?':
```

Comment:
please allow `=` as well so we can fully support URL query strings

Reply:
`=` is a separator actually, so it cannot be part of namespace element name

Reply:
then there's no point for allowing & and ? - please drop them, we can always add them back if needed

Reply:
done